File: Java/Guias/guia3/ej9_Figures/Circle.java
package Guias.guia3.ej9_Figures;

public class Circle extends Ellipse{
    Circle(Point center, double radius){
        super(center, 2*radius, 2*radius);   //asigno a ambos ejes el diametro del circulo
    }

    @Override
    public String toString(){
        return "Circle:\n\tCenter: %s\n\tRadius: %.2f".formatted(centerPoint, mayAxis/2);
    }
}

File: RUBY/Guias/Guia8/Ej4_Figures/circle.rb
# frozen_string_literal: true

require_relative 'ellipse'
class Circle < Ellipse
  def initialize(center, radius)
    super(center, radius*2, radius*2)
  end

  def to_s
    "Circulo [Centro: #{@center}, Radio: #{@ax1/2}]"
  end
end
